multiapi-engine/src/test/resources/asyncapigenerator/testCustomValidators/customvalidator/MinItemsValidator.java

```diff
@@ -0,0 +1,30 @@
+package com.sngular.scsplugin.customvalidator.model.event.customvalidator;
+
+import java.util.List;
+import java.util.Objects;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class MinItemsValidator implements ConstraintValidator<MinItems, List<?>> {
+
+  private int minimum;
+
+  @Override
+  public void initialize(MinItems constraintAnnotation) {
+    ConstraintValidator.super.initialize(constraintAnnotation);
+    this.minimum = constraintAnnotation.minimum();
+  }
+
+  @Override
+  public boolean isValid(List<?> value, ConstraintValidatorContext context) {
+    if (Objects.isNull(value))
```

Comment:
```
return Objects.isNull(value) || !(value.size() < this.minimum);
```

Reply:
Done and also changed for the Max Items validator